src/Core/Models/Request/SendRequest.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using Bit.Core.Enums;
+using Bit.Core.Models.Api;
+using Bit.Core.Models.Domain;
+
+namespace Bit.Core.Models.Request
+{
+    public class SendRequest
+    {
+        public SendType Type { get; set; }
+        public string Name { get; set; }
+        public string Notes { get; set; }
+        public string Key { get; set; }
+        public int? MaxAccessCount { get; set; }
+        public DateTime? ExpirationDate { get; set; }
+        public DateTime DeletionDate { get; set; }
+        public SendTextApi Text { get; set; }
+        public SendFileApi File { get; set; }
+        public string Password { get; set; }
+        public bool Disabled { get; set; }
+
+        public SendRequest(Send send)
+        {
+            Type = send.Type;
+            Name = send.Name?.EncryptedString;
+            Notes = send.Notes?.EncryptedString;
+            MaxAccessCount = send.MaxAccessCount;
+            ExpirationDate = send.ExpirationDate;
+            DeletionDate = send.DeletionDate;
+            Key = send.Key?.EncryptedString;
+            Password = send.Password;
+            Disabled = send.Disabled;
+
+            switch (Type)
+            {
+                case SendType.Text:
+                    Text = new SendTextApi
+                    {
+                        Text = send.Text?.Text?.EncryptedString,
+                        Hidden = send.Text.Hidden
+                    };
+                    break;
+                case SendType.File:
+                    File = new SendFileApi()
```

Comment:
Like the constructor call for `SendTextApi` above on line 37, remove the `()` when using a simple object creation construct.